pkg/minikube/command/ssh_runner.go

```diff
@@ -189,6 +196,49 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	return g.Wait()
 }
 
+func (s *SSHRunner) fileExistsInVM(f assets.CopyableFile, dst string) bool {
+	sess, err := s.c.NewSession()
+	if err != nil {
+		return false
```

Comment:
Masking the error like this could have unintended consequences. The guidance is to always log an error, or return it.

Given how much error handling happens here, I feel like changing the function to return either just an error or a bool and an error would make sense.

Reply:
Makes sense, done